README.md

```diff
@@ -9,51 +9,50 @@ PermaplanT is an app for
 
 The overall goals of the app are to enable planning:
 
-1. for a good yield, while
-2. providing a diverse and fully functioning ecosystem, and
-3. creating living spaces for both animals & humans.
+1. vital growth of delicious edible crops, while
+2. providing a diverse and functioning ecosystem, and
+3. creating outdoor living spaces for both animals & humans.
 
 To achieve these goals the app provides, e.g.:
 
 - visual indications for what:
-  1. makes sense according to natural constraints
-  2. is practical
+  1. makes sense according to known and easily measurable natural constraints
+  2. is practical for the gardener
   3. enhances diversity and supports the ecosystem
 - suggestions for:
-  - increasing diversity (not only what was used recently)
-  - what fits well in the time of the year
+  - what is seasonal
+  - increasing diversity
 
 ## Main Features
 
 - users can have any number of maps
 - users can work on the same map together
 - maps have a fixed number of layers (see below)
-- undo/redo
+- undo/redo within current session for changes in the map
```

Comment:
Is this managed locally in the frontend? I assume sessions start when a user visits the website and ends when the user leaves the website

Reply:
I wrote it because I thought it is easier for you to implement. It is also like how I understood what you originally wrote in undo_redo usecase.

From user perspective best would be to be able to undo/redo everything in the last month. But this makes the database quite complicated. But if you know an easy solution, Yvonne would be happy.